docs/guides/build-it-yourself/achieve-compliance/deployment-walkthrough/manual_steps.md

```diff
@@ -0,0 +1,129 @@
+the [AWS docs](https://docs.aws.amazon.com/sns/latest/dg/sns-user-notifications.html) to be notified by Email, Phone,
+or SMS for each of the alerts.
+
+You can also configure an automated system integration if you have a third party alerting system or central dashboard.
+Follow the steps in the [AWS
+docs](https://docs.aws.amazon.com/sns/latest/dg/sns-http-https-endpoint-as-subscriber.html) on how to add an HTTPS endpoint as a subscriber to the alerts.
+
+### 5. Manually maintain buckets to analyze in the `buckets_to_analyze` variable
+
+To set up Macie to analyze the desired S3 buckets, you’ll need to create a **Macie classification job**. Typically,
+you’ll want it to analyze all the buckets in the region. However, the terraform AWS provider does not support specifying
+all the buckets in a region - it requires that an explicit list of buckets be provided. Therefore, you’ll
+need to maintain an explicit list of buckets per region, namely in the variable `buckets_to_analyze`. Please read the
+[documentation](https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/blob/master/modules/security/macie/variables.tf#L21-L30)
+for this variable in order to understand how to structure the list of buckets per region. Once the underlying issue in the
+Terraform AWS provider has been resolved, we will
+[update the Gruntwork Macie module](https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/issues/204)
+to add support for specifying all buckets in a region.
+
+
+### 6. Enable MFA Delete for all S3 buckets
+
+:::caution
+
+We do not recommend having active AWS access keys for the root user, so remember to delete them when you finish this step.
+
+:::
+
+Enabling MFA Delete in your bucket adds another layer of security by requiring MFA in any request to delete a version or change the versioning state of the bucket.
+
+The attribute `mfa_delete` is only used by Terraform to [reflect the current state of the bucket](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#mfa_delete). It is not possible to create a bucket if the `mfa_delete` is `true`, because it needs to be activated [using AWS CLI or the API](https://docs.aws.amazon.com/AmazonS3/latest/userguide/MultiFactorAuthenticationDelete.html).
+
+To make this change [**you need to use the root user of the account**](https://docs.aws.amazon.com/general/latest/gr/root-vs-iam.html#aws_tasks-that-require-root) that owns the bucket, and MFA needs to be enabled.
+
+1. [Configure MFA for the root user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html#id_root-user_manage_mfa), which you did before in this guide.
+2. [Create access keys for the root user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html#id_root-user_manage_add-key).
+
+Authenticated with the root user, use the AWS CLI to enable MFA Delete. If you are using `aws-vault`, it is necessary to use the `--no-session` flag.
```

Comment:
We now have [this KB post ](https://github.com/gruntwork-io/knowledge-base/discussions/647 ) you could consider linking to it so that folks who are curious can learn about why you have to pass the `--no-session` flag.

Reply:
(Otherwise, when I read this the first thing I want to ask is what is that `--no-session` flag doing?)

Reply:
Oh perfect! Thanks